assets/js/langpack/es.json

```diff
@@ -0,0 +1,126 @@
+
+    "Why use Portus":
+    "Por qué Portus",
+
+    "The Docker Hub is a wonderful place, but what if you cannot, or do not wish to push your images to a thrid party registry? You may setup a private instance of the Docker registry, but then everyone in your organization has push and pull privileges over all of your images.":
+    "El Docker Hub es un lugar maravilloso, pero si no puedes, o no deseas hacer push de tus imágenes a registros de terceros? Puedes configurar una instancia privada del registro de Docker, pero entonces cualquiera en tu organización tiene privilegios para hacer push y pull de todas tus imágenes.",
+
+    "How do you keep track of all of the images in your private registry?":
+    "Cómo sigues todas tus imágenes en tu registro privado?",
+
+    "Portus to the rescue!":
+    "Portus al rescate!",
+
+    "Secure":
+    "Seguro",
+
+    "Portus implements the new authorization scheme defined by the latest version of the Docker registry. It allows for fine grained control over all of your images. You decide which users and teams are allowed to push or pull images.":
+    "Portus implementa el nuevo esquema de autorización definido por la última versión del registro de Docker. Permite el control fino de todas tus imágenes. Tú decides qué usuarios y equipos tienen permizo de hacer push o pull de imágenes",
+
+    "Easily manage users with teams":
+    "Maneja usuarios con equipos fácilmente",
+
+    "Map your company organization inside of Portus, define as many teams as you want and add and remove users from them.":
+    "Mapea la organización de to empresa dentro de Portus, define tantos equipos como quieras y agrega o elimina usuarios desde estos.",
+
+    "Teams have three different types of users to allow full granularity:":
+    "Los equipos tienen tres tipos diferentes que permiten granularidad completa:",
+
+    "Viewers":
+    "Visores",
+
+    "Can only pull images.":
+    "Sólo pueden hacer pull de imágenes.",
+
+    "Contributors":
+    "Contribuyentes",
+
+    "Can push and pull images.":
+    "Pueden hacer push y pull de imágenes.",
+
+    "Owners":
+    "Propietarios",
+
+    "Like contributors, but can also add and remove users from the team.":
+    "Igual que los contribuyentes, pero pueden también agregar y eliminar usuarios de equipos.",
+
+    "Search":
+    "Busca",
+
+    "Portus provides an intuitive overview of the contents of your private registry. It also features a search capability to find images even faster.":
+    "Portus provee una visión general intuitiva del contenido de tu registro privado. También dispone de la posibilidad de buscar y encontrar imágenes incluso más rápido",
+
+    "User privileges are constantly taken into account, even when browsing the contents of the repository or when performing searches.":
+    "Los privilegios de usuarios son constantemente tenidos en cuenta, incluso navegando contenido del repositorio o realizando una búsqueda.",
```

Comment:
Re-phrase it like: "Portus tiene en cuenta constantemente los privilegios de los usuarios, incluso navegando el contenido"... also notice the "el"